plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineWebXmlValidatorTest.java

```diff
@@ -55,14 +55,14 @@ public static void setUp() throws CoreException {
     createFolders(project, new Path("src/main/webapp/WEB-INF"));
     IFile webXml = project.getFile("src/main/webapp/WEB-INF/web.xml");
     webXml.create(new ByteArrayInputStream(new byte[0]), true, null);
-    resource = webXml;
+    webXmlFile = webXml;
   }
 
 
   @After
   public void tearDown() throws CoreException {
-    if (resource != null) {
-      resource.deleteMarkers(IMarker.PROBLEM, true, IResource.DEPTH_ZERO);
+    if (webXmlFile != null) {
```

Comment:
Would it also be better to make project non-static and use Before instead of BeforeClass? I wasn't sure about that because then I'm creating a new project for every test, but there aren't that many tests so it's probably fine?